### PR TITLE
Ryanyae gallery2

### PR DESCRIPTION
**Pull request by ryanyae**
## Description

Made the gallery dynamic with a new implementation.

NOTE: 
for testing reasons the gallery for correlation has some made up cards for the purpose of testing
## Updates

At this point in time, there are only 2 cards, the default gallery item card, and the no-photo-test card for testimonies with no photos.

## Checklist

- [x] Ran `npm run lint:fix` to format code
- [x] Requested at least one reviewer
- [x] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by ryanyae** [comment]
[home page gallery (no text) refactor](https://trello.com/c/5RB4ZI0K/167-home-page-gallery-no-text-refactor)

File: src/components/ProjectGallery2/ProjectGallery2.tsx [hunk 1]
```diff
@@ -0,0 +1,214 @@
+import { useEffect, useState, useCallback } from 'react';
+import './ProjectGallery2.css';
+import { GALLERY_ITEM } from '@statics/projectsV2';
+import { ImQuotesLeft, ImQuotesRight } from "react-icons/im";
+import { IoIosArrowDropleft, IoIosArrowDropright } from "react-icons/io";
+
+
+const AUTO_TIME = 10; // Seconds.
```
Comment (ElisaHu1) [inline src/components/ProjectGallery2/ProjectGallery2.tsx:8]: why we are setting this to 10 second? I remember in the laptop view we can see the transition faster?

Comment (ryanyae) [reply src/components/ProjectGallery2/ProjectGallery2.tsx:8]: I can definetly change it, I just didn't know how long we wanted it to be. Do you have any suggestions for the time, 5s?

Comment (ElisaHu1) [reply src/components/ProjectGallery2/ProjectGallery2.tsx:8]: 5s sounds good to me!

Comment (ryanyae) [reply src/components/ProjectGallery2/ProjectGallery2.tsx]: Done!

**Review by ElisaHu1 — APPROVED** [review]
looks great!

**Pull request merged**